### DISCUSSION TITLE
download speed is not accurate

### DISCUSSION THREAD

**Discussion by twisthead** [prateekmedia/netspeedsimplified#21] [Q&A] [answered]
I think the download speed is not accurate. My network speed never goes up that much the highest I could get is ~2.5 megs, but the screenshot attached it showing 7 mbps. Is this an issue or I am at wrong?

![Screenshot from 2021-05-08 02-09-38](https://user-images.githubusercontent.com/75267529/117507459-a6a10580-afa4-11eb-932c-2220182af625.png)

**Comment by prateekmedia** [comment, marked as answer]
Hi @twisthead,

You are in bits/s mode or you can say small b /s mode, if you need accurate speed(which is in Bytes / s) then you need to click on the speed indicator once more.

Summary:
b/s => not accurate bits per sec
B/s => accurate bytes per sec

Also you need to read the read it is clearly stated in that about the modes of the speed indicator.

If you are still confused then go to extensions app and click on gear icon for netspeedsimplified and then set Mode to Mode 4, thats it.

**Reply by twisthead** [reply]
thanks for the reply I appreciate the support. amazing extention.
